Formula/clusterawsadm.rb

```diff
@@ -0,0 +1,26 @@
+class Clusterawsadm < Formula
+  desc "Home for bootstrapping, AMI, EKS, and other helpers in Cluster API Provider AWS"
+  homepage "https://cluster-api-aws.sigs.k8s.io/clusterawsadm/clusterawsadm.html"
+  url "https://github.com/kubernetes-sigs/cluster-api-provider-aws.git",
+      tag:      "v2.0.2",
+      revision: "28bc9b8756d6d7f73038604038ddb3ccf4b22396"
+  license "Apache-2.0"
+  head "https://github.com/kubernetes-sigs/cluster-api-provider-aws.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    # Don't dirty the git tree
+    rm_rf ".brew_home"
+
+    system "make", "clusterawsadm"
+    bin.install Dir["bin/*"]
+
+    generate_completions_from_executable(bin/"clusterawsadm", "completion", shells: [:bash, :zsh])
```

Comment:
with this, fish completion would be supported
```suggestion
    generate_completions_from_executable(bin/"clusterawsadm", "completion")
```